errors.js

```diff
@@ -0,0 +1,23 @@
+const defineError = require('ut-error').define;
```

Comment:
We are trying to remove direct references to ut-error, so that we eliminate risk of multiple ut-error being installed. There are now methods like port.defineError and port.getError and also bus.errors that holds all methods of ut-error. Can you check if you can use these, instead? If needed, @stambata can assist for any improvements within the port or bus, as we have this as an initiative in the current release.